.eslintrc

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I would like to not include this in the squashed commit for this PR. In particular, I think what `eslint` rules we end up using should be up to discussion (I probably would prefer to make them stricter, and include [`eslint-config-airbnb`](https://github.com/airbnb/javascript)). But for now, it might be cleaner to not include this in the merge commit?